conf/stats.conf.example

```diff
@@ -0,0 +1,23 @@
+[metric 'sample metric count']
+type=mysql_query
+statsd_type=count
+statsd_ns=source.statsd.test.count
+# mysql_query must return a single column with a single row
```

Comment:
Single column seems to be true but your code seems to handle multiple rows from what I can tell.

Lets make it consistent, whether its by limiting it to a single row or by adjusting the comments here